content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,464 @@
+
+
+## Install scanner updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry. It is required to install the scanner updater AFTER a successful initial NeuVector deployment.
+
+Download [scanner.yaml](./scanner.yaml).
+Execute the following command in a Cloud Shell or Node which has access to kubectl.
+
+Install scanner:
+```
+kubectl apply  -f scanner.yaml
+```
+The output should be similar to the example below:
+
+```
+deployment.apps/neuvector-scanner-pod created
+cronjob.batch/neuvector-updater-pod created
+```
+
+Check the scanner pods are running:
+```
+kubectl get pods -n neuvector
+```
+
+The output should be similar to the example below:
+```
+NAME                                       READY   STATUS      RESTARTS   AGE
+neuvector-controller-pod-5687c4f89-5cdb6   1/1     Running     0          21h
+neuvector-controller-pod-5687c4f89-dqqmh   1/1     Running     0          21h
+neuvector-controller-pod-5687c4f89-jgnp9   1/1     Running     0          21h
+neuvector-csp-pod-6c6d945488-46tnj         1/1     Running     0          21h
+neuvector-enforcer-pod-28p52               1/1     Running     0          21h
+neuvector-manager-pod-5c75bd44bb-r867s     1/1     Running     0          21h
+neuvector-scanner-pod-9489c5d78-mzthg      1/1     Running     0          7s
+neuvector-scanner-pod-9489c5d78-rzjcz      1/1     Running     0          9s
+neuvector-scanner-pod-9489c5d78-v5xrq      1/1     Running     0          10s
+neuvector-updater-pod-28294240-lpshs       0/1     Completed   0          10m
+neuvector-updater-pod-28294245-xrk4n       0/1     Completed   0          5m11s
+neuvector-updater-pod-28294250-hqpcg       0/1     Completed   0          11s
+```
+
+
+## Log into the NeuVector console
+
+The manager service type was set to Load Balancer during install. An external host name has been assigned for accessing the NeuVector console. By default, this URL is accessible from the Internet. However, your organization may have placed additional restrictions on external access to your cluster. To retrieve details about the load balancer, execute the following command in Cloud Shell:
+
+
+```
+kubectl get svc -n neuvector neuvector-service-webui
+```
+
+The full dashboard URL can be retrieved with the following commands:
+
+```
+SERVICE_IP=$(kubectl get svc --namespace neuvector neuvector-service-webui -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+echo https://$SERVICE_IP:8443
+```
+
+This will print out a URL that looks like 
+
+```
+https://10.20.10.1:8443
+```
+
+This URL provides access to the NeuVector console which is running by default on port 8443. Open a browser window, paste the URL.
+The NeuVector login will appear, Log in with the credentials you have set for an admin user during offer creation.
+
+## Navigating the console
+
+Once logged in, you can begin to [navigate and configure NeuVector] (https://open-docs.neuvector.com/navigation/navigation).
```

Comment:
I think there's an extra space here, I assume "navigate and configure NeuVector" is meant to be a hyperlink

Reply:
@jeremy-moffitt . After I sync up with keith, I will update the offer name later.